File: 준영/LV1/2016년.js
function solution(a, b) {
  const week = ['FRI','SAT','SUN','MON','TUE','WED','THU']
  const month = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
  var answer = ''
  var result = 0

  for (let i = 0; i < a-1; i++) {
    result += month[i]
  }

  result += b
  answer = week[(result-1)%7]
  
  return answer;
}

File: 준영/프로그래머스/LV1/2016년.py
# 16년 1월 1일이 금요일
def solution(a, b):
    week = ['FRI','SAT','SUN','MON','TUE','WED','THU']
    month = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    answer = ''
    result = 0
    for i in range(a-1):
        result += month[i]
    
    result += b
    answer = week[(result-1)%7]
    
    return answer

solution(5, 24)